build-package.sh

```diff
@@ -169,9 +169,25 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 # Generate jazzy docs (macOS)
-if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
-    sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+#if [ "${docs}" == "true" ] && [ "$(uname)" == "Darwin" ]; then
+#sourceScript "${projectFolder}/Package-Builder/jazzy.sh"
+#fi
+
+# Generate jazzy docs (macOS) where the 'documentation' tag exists on the issue label
+if [ "$(uname)" == "darwin"* ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
```

Comment:
Do we only generate docs in PRs? I thought we generated once the PR is merged into master then travis runs? Could you explain the flow?

Reply:
It's only in PRs, where it checks for labels called 'jazzy-doc', any time Package Builder is run, it checks through to also see whether the tag exists on the respective PR and then generates docs.

Reply:
okay, so why do we check for `master` in jazzy.sh?

Reply:
That's a valid point, that was left over from when it wasn't only done on PRs, will remove that.

Reply:
So we generate the docs in the PR and then we push to master the docs? before merging the  PR? I think the better flow would be this:
- Merge the PR
- Travis Runs on Master
- Generate the docs.
- Push to master
What do you think?

Reply:
Or we could create another PR, just for the docs?